src/qref/verification.py

```diff
@@ -33,7 +34,7 @@ def __bool__(self) -> bool:
         return self.is_valid
 
 
-def verify_topology(routine: Union[SchemaV1, RoutineV1]) -> TopologyVerificationOutput:
+def verify_topology(routine: SchemaV1 | RoutineV1) -> TopologyVerificationOutput:
```

Comment:
Why not use `accepts_all_qref_types` here?

Reply:
Great idea, fixed!